packages/sdk/src/server/__tests__/server.test.ts

```diff
@@ -1,12 +1,38 @@
 import {
-  ServerClient, createClient,
+  ServerClient, createClient, HTTPAuthType,
 } from "../index";
 import fetchMock from "jest-fetch-mock";
 import { ClientCredentials } from "simple-oauth2";
 
-describe("ServerClient", () => {
-  let client: ServerClient;
+let client: ServerClient;
```

Comment:
Moving this up to the `beforeEach` block. Previously I'd been duplicating the client in many tests (see removal below)